Jbarget/oxymore
# So/fashion editorial

### What changes have you made?

- replaced the images with new images 
- passed a new prop to the `EditorialImage` component called `GridColum` so that landscape images take two columns
- Originally intended to include the `Scrollback` component but couldn´t get it to work for some strange reason so I've taken it out

### Which issue(s) does this PR fix?

Fixes #14
Fixes #237 

### Screenshots (if there are design changes)

### How to test
- Set the timer back to a date in the past and the content shoud display 
- Check that the landscape images display on two columns
- Check that the portrait images display on one column
